app/webpacker/components/ContactsPage/ContactForm.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { useMemo, useRef, useState } from 'react';
+import {
+  Form, FormGroup, FormField, Button, Radio, Message,
+} from 'semantic-ui-react';
+import ReCAPTCHA from 'react-google-recaptcha';
+import { contactUrl } from '../../lib/requests/routes.js.erb';
+import useInputState from '../../lib/hooks/useInputState';
+import useSaveAction from '../../lib/hooks/useSaveAction';
+import I18n from '../../lib/i18n';
+import { RECAPTCHA_PUBLIC_KEY } from '../../lib/wca-data.js.erb';
+import UserData from './SubForms/UserData';
+import Loading from '../Requests/Loading';
+import Wct from './SubForms/Wct';
+import Competition from './SubForms/Competition';
+
+const CONTACT_TYPES = [
+  'competition',
+  'competitions_in_general',
+  'results_team',
+  'wca_id_or_profile',
+  'media',
+  'software',
+  'different',
+];
+
+const CONTACT_TYPES_MAP = CONTACT_TYPES.reduce((accumulator, contactType) => {
+  accumulator[contactType] = contactType;
+  return accumulator;
+}, {});
```

Comment:
```suggestion
const CONTACT_TYPES_MAP = _.keyBy(CONTACT_TYPES);
```

Reply:
Normally, you would have to pass in a second argument, which is a callback that determines the key to use. But if you skip that argument, it will use "identity" function as a default: https://lodash.com/docs/4.17.15#keyBy